server/portal/posts.py

```diff
@@ -72,9 +80,17 @@ def get_posts():
     return jsonify({'posts': json_arr})
 
 
+"""
+Endpoint: /portal/posts/<post_id>
+HTTP Methods: GET
+Response Formats: JSON
+Parameters: account_id
+
+Returns post information by post ID
+"""
 @app.route('/portal/post/<int:post_id>', methods=['GET'])
 def get_post(post_id):
-    account_id = request.args.get('account')
+    account_id = request.args.get('account_id')
```

Comment:
Will need to update penn-mobile-portal after making this change.

Reply:
Reversed these changes to their initial state.